CHANGELOG.md

```diff
@@ -1,17 +1,20 @@
 # Changelog
 
-## Next Release
+## 22.11.0
 ### Breaking Changes
-- Slashing protection imports will now only fail for an individual validator instead for all validators allowing partial import if there is valid and invalid data.
+- Slashing protection imports will now only fail for an individual validator instead for all validators allowing partial 
+import if there is valid and invalid data.
 
 ### Features Added
 - Introduced cli option to specify Hikari configuration for pruning database connection [#661](https://github.com/ConsenSys/web3signer/issues/661)
-- Better database pruning default values: Pruning enabled by default with `slashing-protection-pruning-epochs-to-keep = 250`, `slashing-protection-pruning-at-boot-enabled = false` and `slashing-protection-pruning-interval = 12`
+- Better database pruning default values: Pruning enabled by default with 
```

Comment:
Is there something weird going on with the line breaks here or is it just the diff?

Reply:
no, I made line breaks so that its easier to use the source file directly.